airbyte-integrations/connectors/source-stripe/setup.py

```diff
@@ -30,5 +30,5 @@
     author_email="contact@airbyte.io",
     packages=find_packages(),
     package_data={"": ["*.json", "schemas/*.json", "schemas/shared/*.json"]},
-    install_requires=["airbyte-protocol", "base-python", "stripe"],
+    install_requires=["airbyte-cdk-test==0.1.0rc2", "stripe"],
```

Comment:
TODO update this to `airbyte-cdk` once davin stops domain-squatting the `airbyte-cdk` package from his personal PyPi account 🤣